nrf52840_layout.ld

```diff
@@ -0,0 +1,17 @@
+/* Layout for the nRF52840-DK, usable by the examples in this repository. */
+
+MEMORY {
+  /* The application region is 64 bytes (0x40) */
+  FLASH (rx) : ORIGIN = 0x00030040, LENGTH = 0x000CFFC0
+  SRAM (rwx) : ORIGIN = 0x20004000, LENGTH = 62K
```

Comment:
I don't think the SRAM length is correct. Isn't there more memory than that on the nRF52840-DK?